crates/re_types/definitions/rerun/blueprint/archetypes/panel_blueprint.fbs

```diff
@@ -0,0 +1,21 @@
+include "fbs/attributes.fbs";
+include "rust/attributes.fbs";
+
+namespace rerun.blueprint.archetypes;
+
+// ---
+
+/// Shared state for the 3 collapsible panels.
+table PanelBlueprint (
+    "attr.rerun.scope": "blueprint",
+    "attr.rust.derive": "Default",
+    "attr.rust.override_crate": "re_viewport"
+) {
+    // --- Required ---
+
+    // --- Optional ---
+    /// Whether or not the panel is expanded.
+    expanded: rerun.blueprint.components.PanelExpanded ("attr.rerun.component_optional", nullable, order: 1000);
+
+    // TODO(jleibs): Add a float to track how expanded the panel is.
```

Comment:
You mean how wide it is in relation to the screen or how far the animation for expanding has progressed?
The former makes sense to me, but the latter doesn't really imho; we should leave ephemeral ui animations in the egui state.

(Was wondering for a bit if that means that `expanded` is a float, but keeping it separate makes sense because that's how the ui has to work as well)

Reply:
Yeah, the target width/height, not the animation state.  I had the same thought on combining them into a single field and then realized we wanted to return to the set width on toggle.